src/platform/graphics/constants.js

```diff
@@ -946,8 +946,95 @@ export const PIXELFORMAT_R8 = 52;
  */
 export const PIXELFORMAT_RG8 = 53;
 
-// map of engine PIXELFORMAT_*** enums to information about the format
-// .srgb: true if sampling the format returns value in sRGB space, otherwise in linear space
+/**
+ * Format equivalent to {@link PIXELFORMAT_DXT1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_DXT1_SRGB = 54;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_DXT3} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_DXT3_SRGB = 55;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_DXT5} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_DXT5_SRGB = 56;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_2BPP_RGB_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_2BPP_SRGB_1 = 57;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_2BPP_RGBA_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_2BPP_SRGBA_1 = 58;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_4BPP_RGB_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_4BPP_SRGB_1 = 59;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_4BPP_RGBA_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_4BPP_SRGBA_1 = 60;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_ETC2_RGB} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_ETC2_SRGB = 61;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_ETC2_RGBA} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_ETC2_SRGBA = 62;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_ASTC_4x4} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_ASTC_4x4_SRGB = 63;
+
+/**
+ * Information about pixel formats.
+ *
+ * srgb: true if sampling the format returns value in sRGB space, otherwise in linear space
+ * linearFormat: the corresponding linear format for sRGB formats (sampled in linear space)
```

Comment:
Erm, it's the other way around. RGBA8 is a linear format, stored and sampled in linear space.

The pixel formats you're mapping to here are the sRGB versions. This should probably be called `srgbFormat` or similar.

Also wondering why the linear formats have been flagged as 'srgb: true' (in a previous PR)? `PIXELFORMAT_DXT1` is definitely not srgb and neither are the others.

Reply:
I updated the comments to better explain the meaning, please review this